src/applications/vaos/appointment-list/pages/EpsAppointmentDetailsPage/EpsAppointmentDetailCard.jsx

```diff
@@ -47,21 +44,15 @@ export default function EpsAppointmentDetailCard({
         <span data-dd-privacy="mask">{pageTitle}</span>
       </h1>
       <Section heading="When">
-        <AppointmentDate
-          date={appointment.start}
+        <AppointmentDateTime
+          start={appointment.start}
           timezone={appointment.provider.location.timezone}
+          calendarData={calendarData}
+          calendarButton={
```

Comment:
Could this prop be more descriptive? 
```suggestion
          addToCalendarButton={
```